config/routes.rb

```diff
@@ -170,6 +170,8 @@
     end
   end
 
+  get '/map_view_details' => 'families#map_view_details'
+  get '/directory_map' => 'families#directory_map', via: :get, as: :directory_map
```

Comment:
This definitely needs its own controller. I'm thinking we can call it `directory_maps_controller`.